app/src/main/java/com/framgia/wsm/screen/changepassword/ChangePasswordViewModel.java

```diff
@@ -22,4 +55,105 @@ public void onStart() {
     public void onStop() {
         mPresenter.onStop();
     }
+
+    @Override
+    public void onChangePasswordError(BaseException e) {
+        mNavigator.showToast(e.getMessage());
+    }
+
+    @Override
+    public void onInputCurrentPasswordError(String message) {
+        mCurrentPasswordError = message;
+        notifyPropertyChanged(BR.currentPasswordError);
+    }
+
+    @Override
+    public void onInputNewPasswordError(String message) {
+        mNewPasswordError = message;
+        notifyPropertyChanged(BR.newPasswordError);
+    }
+
+    @Override
+    public void onInputConfirmPasswordError(String message) {
+        mConfirmPasswordError = message;
+        notifyPropertyChanged(BR.confirmPasswordError);
+    }
+
+    @Bindable
+    public String getCurrentPasswordError() {
+        return mCurrentPasswordError;
+    }
+
+    public void setCurrentPasswordError(String currentPasswordError) {
+        mCurrentPasswordError = currentPasswordError;
+        notifyPropertyChanged(BR.currentPasswordError);
+    }
+
+    @Bindable
+    public String getNewPasswordError() {
+        return mNewPasswordError;
+    }
+
+    public void setNewPasswordError(String newPasswordError) {
+        mNewPasswordError = newPasswordError;
+        notifyPropertyChanged(BR.newPasswordError);
+    }
+
+    @Bindable
+    public String getConfirmPasswordError() {
+        return mConfirmPasswordError;
+    }
+
+    public void setConfirmPasswordError(String confirmPasswordError) {
+        mConfirmPasswordError = confirmPasswordError;
+        notifyPropertyChanged(BR.confirmPasswordError);
+    }
+
+    @Bindable
+    public String getCurrentPassword() {
+        return mCurrentPassword;
+    }
+
+    public void setCurrentPassword(String currentPassword) {
+        mCurrentPassword = currentPassword;
+        notifyPropertyChanged(BR.currentPassword);
+    }
+
+    @Bindable
+    public static String getsNewPassword() {
+        return sNewPassword;
+    }
+
+    public static void setsNewPassword(String sNewPassword) {
+        ChangePasswordViewModel.sNewPassword = sNewPassword;
+    }
+
+    @Bindable
+    public String getConfirmPassword() {
+        return mConfirmPassword;
+    }
+
+    public void setConfirmPassword(String confirmPassword) {
+        mConfirmPassword = confirmPassword;
+    }
+
+    public void validateNewPassword(String newPassWord) {
+        mPresenter.validateNewPasswordInput(newPassWord);
+        mPresenter.validateConfirmPasswordInput(mConfirmPassword);
+        sNewPassword = newPassWord;
```

Comment:
ERROR: [STYLE][ST_WRITE_TO_STATIC_FROM_INSTANCE_METHOD] Write to static field com.framgia.wsm.screen.changepassword.ChangePasswordViewModel.sNewPassword from instance method com.framgia.wsm.screen.changepassword.ChangePasswordViewModel.validateNewPassword(String)
http://findbugs.sourceforge.net/bugDescriptions.html#ST_WRITE_TO_STATIC_FROM_INSTANCE_METHOD